tutorials/darcy_thermo_mech/doc/config.yml

```diff
@@ -43,6 +43,7 @@ Extensions:
         macros:
             \vec: "\\bar" # TODO: \vec doesn't render
             \pf: "\\frac{\\partial #1}{\\partial #2}"
+            \pd: "\\frac{\\partial #1}{\\partial #2}"
```

Comment:
Not quite. I just noticed that in a few steps that `\pd` was undefined. I could just replace `\pd` -> `\pf`. what do you think?

Reply:
I think we should replace the two instances I see of `\pd` with `\pf` in the workshop docs to stay consistent with the established config macro for `darcy_thermo_mech` rather than making a new one. The irritating thing is that we use both of these macros all over the place to mean the same thing (even in some tests). Should probably take the time to consolidate at some point, but that's beyond the scope of this.